src/scenes/TableScene.ts

```diff
@@ -86,6 +91,18 @@ export class TableScene extends Scene {
         text.init(this.logic, ware);
     }
 
+    private addQuantityButton(x: number, y: number, quantity: number) {
+        this.quantityButton = new QuantityButton(
```

Comment:
```suggestion
        const quantityButton = new QuantityButton(
```

Reply:
There is no need to make this a data field. In fact, you overwrite the data field twice. Also delete the data field.